tables/leaf.py

```diff
@@ -4,7 +4,11 @@
 import math
 
 import numpy as np
-import cpuinfo
+try:
+    import cpuinfo
+    cpuinfo_available = True
+except ImportError:
+    cpuinfo_available = False
```

Comment:
If you don't mind, I would suggest to reduce the patch footprint as follows:

```diff
- import cpuinfo
+ try:
+     import cpuinfo
+     cpu_info = cpuinfo.get_cpu_info()
+ except ImportError:
+     cpu_info = {}
```

then the other changes can just be dropped.

Reply:
Much nicer, thank you!